public/src/main.tsx

```diff
@@ -144,111 +149,109 @@ const AppRouter = () => {
   );
 
   return (
-    <ThemeProvider theme={getTheme()}>
```

Comment:
`ThemeProvider` is now outside of this component, because we're not allowed to `useStyles` (above) before